masterlist.yaml

```diff
@@ -8680,37 +8782,131 @@ plugins:
-      - 'NRM Dragonborn - Requiem Patch.esp'
+      - 'KFR-Kryptopyr''sFixesReqtified.esp'
+  - name: 'COR-Campfire.esp'
+    req:
+      - 'COR-CraftingOverhaulReqtified.esp'
+  - name: 'COR-WetAndCold.esp'
+    req:
+      - 'COR-CraftingOverhaulReqtified.esp'
+  - name: 'Open Face Guard Helmets.esp'
+    after:
+      - 'KFR-Kryptopyr''sFixesReqtified.esp'
+  - name: 'Requiem_Minor_Arcana.esp'
+    after:
+      - 'KFR-Kryptopyr''sFixesReqtified.esp'
+      - 'COR-CraftingOverhaulReqtified.esp'
+      - 'Requiem - Immersive Spells.esp'
+      - 'Open Face Guard Helmets.esp'
+      - 'Requiem - Civil War Overhaul Patch.esp'
+  # Must be loaded directly after Minor Arcana
+  - name: 'Requiem - Minor Arcana - USLEEP patch.esp'
+    priority: -127
+  - name: 'Requiem - ESF Companions.esp'
+    after:
+      - 'Requiem_Minor_Arcana.esp'
+  - name: 'Requiem - WRF_MinorArcana_MortalEnemies_TG.esp'
+    after:
+      - 'Requiem_Minor_Arcana.esp'
+  - name: 'Requiem - Immersive Creatures.esp'
+    after:
+      - 'Fozars_Dragonborn_-_Requiem_Patch.esp'
+  - name: 'Requiem - Immersive Horses.esp'
+    after:
+      - 'REQMOD.esp'
+  - name: 'Requiem - Immersive Horses - SkyTEST.esp'
+    req:
+      - 'Requiem - SkyTEST.esp'
+    after:
+      - 'REQMOD.esp'
+  - name: 'Requiem - Craftable Horse Barding.esp'
+    after:
+      - 'Requiem - Immersive Horses.esp'
+      - 'Requiem - Immersive Horses - SkyTEST.esp'
+  - name: 'Requiem - WAF+Clothing and Clutter Fixes Patch.esp'
+    inc:
+      - 'KFR-Kryptopyr''sFixesReqtified.esp")'
+  - name: 'Requiem - WAF+CCF+TrueWeapons Patch.esp'
+    inc:
+      - 'KFR-Kryptopyr''sFixesReqtified.esp")'
+# Mods that must be loaded after Requiem
+  - name: 'Bounty Gold.esp'
+    after:
+      - 'Requiem.esp'
+  - name: 'dD.?- Realistic Ragdoll Force.*\.esp'
+    after:
+      - 'Requiem.esp'
+    msg:
+      - type: say
+        content:
+          - lang: en
+            text: 'Ragdolls will conflict with other mods that change the skeleton.nif. Check [ Custom Skeleton Replacers](http://www.nexusmods.com/skyrim/mods/16712) for compatibly with other skeleton.nif.'
+          - lang: ru
+            text: 'Конфликтует с другими модами, меняющими файл skeleton.nif. Смотрите страницу [ Custom Skeleton Replacers](http://www.nexusmods.com/skyrim/mods/16712), для поддержки других версий skeleton.nif или попробуйте использовать [XP32 Maximum Skeleton Extended - XPMSE](http://www.nexusmods.com/skyrim/mods/68000).'
+  - name: 'XPMSE.esp'
+    after:
+      - 'Requiem.esp'
+  - name: 'RealisticRoomRental(-Basic)?.esp'
+    after:
+      - 'Requiem.esp'
+  - name: 'Unlimited Training.esp'
+    after:
+      - 'Requiem.esp'
+  - name: 'WerewolfPerksExpanded.esp'
+    after:
+      - 'Requiem.esp'
+  - name: 'No Vendor or Loot Spells - Riverwood Trader Requiem.esp'
+    req:
+      - 'Requiem.esp'
+  # other mods that must be loaded after Requiem but are listed somewhere else in LOOT's rules:
+  # - RealisticWaterTwo.esp
```

Comment:
I added that comment so that all mods that must be loaded after Requiem are listed in one section as there is no way to search for `after` `Requiem.esp`. If you really think that's just clutter I'll remove it.